components/post_view/acknowledgements/index.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/common';
+import {getHasReactions, getPostAcknowledgementsWithProfiles} from 'mattermost-redux/selectors/entities/posts';
+
+import {GlobalState} from 'types/store';
+
+import {Post} from '@mattermost/types/posts';
+
+import PostAcknowledgements from './post_acknowledgements';
+
+type OwnProps = {
+    postId: Post['id'];
+};
+
+function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
+    const currentUserId = getCurrentUserId(state);
+    const hasReactions = getHasReactions(state, ownProps.postId);
+    const list = getPostAcknowledgementsWithProfiles(state, ownProps.postId);
+
+    let acknowledgedAt = 0;
```

Comment:
Since we're already passing in the current user ID and the acknowledgement list into the component, we could consider doing this in there as well instead of having it in the `mapStateToProps`